### PR TITLE
super-sub script tags added.

### PR DESCRIPTION
**Pull request by kyuukestu**
BBCode additions for superscript e.g. [sup] Content [/sup] and subscript e.g. [sub] Content [/sub].

### CODE REVIEW

File: bbcode-src/tags/subscript.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+/**
+ * @file Adds subscript to bbcode
+ * @example [sub]content[/sub]
+ */
+
+const sub = (node) => {
+  return toNode(
+    "sub",
+    { style: "font-variant-position: sub;" },
```
Comment (Alteras1) [inline bbcode-src/tags/subscript.js:11]: i thought `<sub>` and `<sup>` already have their own styles applied by the browser? No need for `font-variant-position`

Comment (kyuukestu) [reply bbcode-src/tags/subscript.js:11]: I didn't create them via the HTML tags, would that be preferred? I can change them if that's the case!

Comment (Alteras1) [reply bbcode-src/tags/subscript.js:11]: I meant i didn't think the style parameter is necessary here.

Comment (kyuukestu) [reply bbcode-src/tags/subscript.js]: Alright! I've changed them to use the HTML tags!

File: bbcode-src/tags/subscript.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { toNode } from "../utils/common";
+
+/**
+ * @file Adds subscript to bbcode
+ * @example [sub]content[/sub]
+ */
+
+const sub = () => {
+  return tonode("sub", {}, null);
```
Comment (Alteras1) [inline bbcode-src/tags/subscript.js:9]: looks like a typo, wrong casing. Also dont forget to run `yarn build`

**Review by Alteras1 — APPROVED** [review]

**Pull request merged**